test/e2e/E2EStabilityFeeTreasury.t.sol

```diff
@@ -31,7 +31,7 @@ abstract contract E2EStabilityFeeTreasuryTest is BaseUser, Common {
     // giving 25% of the balance to bob
     uint256 _rad = _coinBalance / 4;
 
-    vm.prank(governor);
+    vm.prank(deployer);
```

Comment:
Why is deployer being swapped with governor in this file?

Reply:
that was a HAI change. previously they used a fake governor for testing, but i guess decided to just use the deployer instead